NBA-Stats-Dashboard/src/components/PlayerStats.jsx

```diff
@@ -1,10 +1,90 @@
-function PlayerStats() {
-    return (
-      <div>
-        <h1>PlayerStats</h1>
+import { useState, useEffect } from "react";
+import axios from "axios";
+import {
+  TopScorersUrl,
+  TopAssistsUrl,
+  TopReboundsUrl,
+} from "../utils/apiEndpoints";
+import "chart.js/auto";
+import { Chart, Tooltip, Legend } from "chart.js";
+import { Bar } from "react-chartjs-2";
+import Dropdown from "./Dropdown";
+
+Chart.register(Tooltip, Legend);
+const options = [
+  {
+    title: "Total Points",
+    url: TopScorersUrl,
+  },
+  {
+    title: "Total Assists",
+    url: TopAssistsUrl,
+  },
+  {
+    title: "Total Rebounds",
+    url: TopReboundsUrl,
+  },
+];
+
+const PlayerStats = () => {
+  const [leagueLeaders, setLeagueLeaders] = useState([]);
+  const [criteria, setCriteria] = useState(options[0]);
+
+  const chartData = {
+    labels: leagueLeaders.slice(0, 5).map((player) => player.player_name),
+    datasets: [
+      {
+        axis: "y",
+        label: "Goals ",
```

Comment:
Why does the label stay the same at "Goals", if the selected criteria is changing "points, rebounds, and assists". This should be able to change dynamically. I would recommend replacing this hardcoded "goals" by using a state variable or some sort of variable that keeps track of the selected criteria. Also, we do not need to worry about field goals / goals in this application because we are just concerned with points, rebounds and assists.